### PR TITLE
Plot train reward and n_updates

### PR DESCRIPTION
**Pull request by rhaps0dy**
- plot train reward
- fix `_n_updates` plotting to correspond to the actual number of updates. That is, taking into account the loop breaking because of too much KL
- use `F.mse_loss` instead of ad-hoc implementation

### CODE REVIEW

**Review by taufeeque9 — APPROVED** [review]
LGTM
